src/containers/Accounts/index.js

```diff
@@ -94,10 +58,6 @@ const Accounts = (props) => {
   )
 }
 
-Accounts.propTypes = {
-  error: PropTypes.number,
-}
-
 Accounts.defaultProps = {
   error: null,
 }
```

Comment:
`connect` logic can be removed since error is no longer used.

Reply:
removed